.changeset/old-ears-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/synapse-guest-module': patch
```

Comment:
@HarHarLinks Shouldn't this be a major change? This PR changes expected behaviour when compared to the previous release, so it can be considered a breaking change, no?

Reply:
Agree